.github/workflows/cf-deploy.yml

```diff
@@ -2,7 +2,7 @@ name: Deploy to bucket
 
 on:
   push:
-    branches: ['green-dev']
+    branches: ['!green-dev']
```

Comment:
@royki I think this won't work:
```
If you define a branch with the ! character, you must also define at least one branch without the ! character. If you only want to exclude branches, use branches-ignore instead.
```
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions